build/storage/scripts/disk_infrastructure.sh

```diff
@@ -95,59 +96,22 @@ print(disk_infrastructure.uuid2base64(device_uuid="$1"))
 EOF
 }
 
-function wait_for_virtio_blk_in_os() {
-    python3 <<- EOF
-from scripts import disk_infrastructure
-
-disk_infrastructure.wait_for_virtio_blk_in_os(
-    timeout=float("${2}")
-)
-EOF
-}
-
-function wait_for_volume_in_os() {
-    python3 <<- EOF
-from scripts import disk_infrastructure
-
-disk_infrastructure.wait_for_volume_in_os(
-    timeout=float("${2}")
-)
-EOF
-}
-
 function create_virtio_blk() {
     python3 <<- EOF
 from scripts import disk_infrastructure
 
 print(
     disk_infrastructure.create_virtio_blk(
         ipu_storage_container_ip="${1}",
-        volume_id="${2}",
-        physical_id="${3}",
-        virtual_id="${4}",
-        hostnqn="${5}",
-        traddr="${6}",
-        trsvcid="${7:-"$DEFAULT_NVME_PORT"}",
-        sma_port=int("${8:-"$DEFAULT_SMA_PORT"}"),
-    )
-)
-EOF
-}
-
-function create_virtio_blk_without_disk_check() {
-    python3 <<- EOF
-from scripts import disk_infrastructure
-
-print(
-    disk_infrastructure.create_virtio_blk_without_disk_check(
-        ipu_storage_container_ip="${1}",
-        volume_id="${2}",
-        physical_id="${3}",
-        virtual_id="${4}",
-        hostnqn="${5}",
-        traddr="${6}",
-        trsvcid="${7:-"$DEFAULT_NVME_PORT"}",
-        sma_port=int("${8:-"$DEFAULT_SMA_PORT"}"),
+        sma_port=int("${2}"),
+        host_target_ip="${3}",
+        host_target_service_port=int("${4}"),
+        volume_id="${5}",
+        physical_id="${6}",
+        virtual_id="${7}",
+        hostnqn="${8}",
+        traddr="${9}",
+        trsvcid="${10:-"$DEFAULT_NVME_PORT"}",
```

Comment:
some are cast explicitly as int and some note?

Reply:
trsvcid is marked as string in sma